source/runner/_tests/DangerRunner.test.ts

```diff
@@ -83,11 +83,40 @@ describe("cleaning Dangerfiles", () => {
     const path = resolve(os.tmpdir(), "fake_dangerfile_1")
     fs.writeFileSync(path, "import { danger, warn, fail, message } from 'danger'")
     updateDangerfile(path)
-    expect(fs.readFileSync(path).toString()).toEqual("// import { danger, warn, fail, message } from 'danger'")
+    expect(fs.readFileSync(path).toString()).toEqual("// Removed import")
   })
 
   it("also handles typescript style imports", () => {
-    const before = "import danger from 'danger'"
-    expect(cleanDangerfile(before)).toEqual("// import danger from 'danger'")
+    const before = `
+import { danger, warn, fail, message } from 'danger'
+import { danger, warn, fail, message } from "danger"
+import { danger, warn, fail, message } from "danger";
+import danger from "danger"
+import danger from 'danger'
+import danger from 'danger';
+`
+    const after = `
+// Removed import
+// Removed import
+// Removed import
+// Removed import
+// Removed import
+// Removed import
+`
+    expect(cleanDangerfile(before)).toEqual(after)
+  })
+
+  it("also handles require style imports", () => {
+        const before = `
+const { danger, warn, fail, message } from 'danger'
```

Comment:
I'm not sure if this is valid syntax/tests what your `it()` description states. The `require` imports would look like:

```js
const { danger, warn, fail, message } = require('danger')
```